radio/src/targets/taranis/board.cpp

```diff
@@ -189,9 +199,14 @@ void boardInit()
   DBGMCU_APB1PeriphConfig(DBGMCU_IWDG_STOP|DBGMCU_TIM1_STOP|DBGMCU_TIM2_STOP|DBGMCU_TIM3_STOP|DBGMCU_TIM6_STOP|DBGMCU_TIM8_STOP|DBGMCU_TIM10_STOP|DBGMCU_TIM13_STOP|DBGMCU_TIM14_STOP, ENABLE);
 #endif
 
+#if defined(EEPROM) && defined(PWR_BUTTON_PRESS)
+  storageReadRadioSettings();
```

Comment:
I think having 2 different paths for reading radio settings is not good from maintenance point of view. We should avoid this if possible